library/std/src/sys/unix/time.rs

```diff
@@ -53,12 +53,6 @@ impl SystemTime {
     }
 }
 
-impl From<libc::timespec> for SystemTime {
```

Comment:
Deleted From implementations on public types need to be at minimum replaced with a TryFrom implementation, or preferably not deleted at all

Reply:
Is `libc::timespec` public? I mean, the libc rust std uses.

Reply:
I removed it cause it's was not even used and was dangerous, AFAIK it's not public but I could be wrong.